package.json

```diff
@@ -106,7 +106,7 @@
 		"@php-wasm/scopes": "^0.9.44",
 		"@php-wasm/universal": "^0.9.44",
 		"@rive-app/react-canvas": "^4.12.0",
-		"@sentry/electron": "^4.17.0",
+		"@sentry/electron": "^4.24.0",
```

Comment:
Again, not something we need to do in this PR, but we should look into [upgrading to the next major version](https://github.com/getsentry/sentry-electron/blob/5.0.0/MIGRATION.md). Are we aware of any potentially breaking issues, @wojtekn?

Reply:
@fredrikekelund yes, we should, but let's do this separately. I want to take the smallest possible step here to fix the security issues reported by Dependabot. We will need to update WP components, Electron and probably more.